packages/web/components/Setup/SetupName.tsx

```diff
@@ -5,23 +5,23 @@ import React from 'react';
 import { ProfileWizardPane } from './ProfileWizardPane';
 import { WizardPaneCallbackProps } from './WizardPane';
 
-export const SetupUsername: React.FC = () => {
-  const field = 'username';
+export const SetupName: React.FC = () => {
+  const field = 'name';
 
   return (
     <ProfileWizardPane
       {...{ field }}
-      title="Username"
-      prompt="What name would you like to use in your MyMeta profile URL?"
+      title="Name"
```

Comment:
I've a feeling just "name" might be somewhat misleading since we do not allow spaces or even uppercase letters to my knowledge. Something like "User Handle" or "MetaGame Handle" ?